uber/models/promo_code.py

```diff
@@ -340,7 +340,9 @@ def normalize_expiration_date(cls, dt):
 
     @property
     def discount_str(self):
-        if not self.discount:
+        if self.discount_type == self._FIXED_DISCOUNT and self.discount == 0:
```

Comment:
This and the change below are to support the art show plugin's use of promo codes to assign agents to art show applications. I'll move these to that plugin.